gladier_xpcs/deployments.py

```diff
@@ -138,11 +138,25 @@ class HannahDeployment(BaseDeployment):
 
     flow_input = {
         'input': {
-            'staging_dir': '/projects/APSDataAnalysis/hannah/gladier_testing/',
+            'staging_dir': '/eagle/projects/APSDataAnalysis/XPCS/hparraga/gladier_testing/',
```

Comment:
We should move that to the XPCS collection and change the paths to `/` without the full project path.

Reply:
I think this should be part of a separate PR. It isn't as easy as swapping the endpoints, the files transferred into the XPCS shared collection appear there without group write permissions, and that makes it impossible to write the corr plots (I think I also saw other permissions issues when playing with this). 

Once we figure out permissions, I think we can switch over. We'll also need to modify the publish tools to remove the extra transfers.